doc/modules/calibration.rst

```diff
@@ -208,3 +208,124 @@ a similar decrease in log-loss.
     .. [5] On the combination of forecast probabilities for
            consecutive precipitation periods. Wea. Forecasting, 5, 640–650.,
            Wilks, D. S., 1990a
+
+
+.. _decision_threshold_calibration:
+
+==============================
```

Comment:
This will mean that the page has two top-level headings. Rather, at the top of the page, create a heading of this level called "Prediction calibration" and then change the heading  level of "Probability Calibration" and "Decision Threshold calibration" to fall under it.